paddle/framework/grad_op_builder.cc

```diff
@@ -13,104 +13,48 @@ express or implied. See the License for the specific language governing
 permissions and limitations under the License. */
 
 #include "paddle/framework/grad_op_builder.h"
-#include "paddle/framework/op_proto.pb.h"
+#include "paddle/framework/framework.pb.h"
 #include "paddle/framework/op_registry.h"
 
 namespace paddle {
 namespace framework {
 
-typedef std::vector<int> Ints;
+class OpRegistry;
```

Comment:
Do not use forward declare in C++;
https://google.github.io/styleguide/cppguide.html#Forward_Declarations